e2e/pages/citizen/createCase/C100/reasonableAdjustments/reasonableAdjustmentsNeedsInCourtPage.ts

```diff
@@ -0,0 +1,198 @@
+        1,
+      ),
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${Selectors.a}:text-is("${ReasonableAdjustmentsNeedsInCourtContent.errorMessageBlank}")`,
+        1,
+      ),
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${Selectors.GovukErrorMessageCitizen}:text-is("${ReasonableAdjustmentsNeedsInCourtContent.errorMessageBlank}")`,
+        1,
+      ),
+    ]);
+    await page.click(CheckListUniqueSelectors.parkingSpace);
+    await page.click(CheckListUniqueSelectors.differentTypeOfChair);
+    await page.click(CheckListUniqueSelectors.other);
+    await Promise.all([
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${Selectors.GovukErrorSummaryTitle}:text-is("${CommonStaticText.errorSummaryTitle}")`,
+        1,
+      ),
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${Selectors.a}:text-is("${ReasonableAdjustmentsNeedsInCourtContent.errorMessageParkingSpace}")`,
+        1,
+      ),
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${Selectors.GovukErrorMessageCitizen}:text-is("${ReasonableAdjustmentsNeedsInCourtContent.errorMessageParkingSpace}")`,
+        1,
+      ),
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${Selectors.a}:text-is("${ReasonableAdjustmentsNeedsInCourtContent.errorMessageTypOfChair}")`,
+        1,
+      ),
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${Selectors.GovukErrorMessageCitizen}:text-is("${ReasonableAdjustmentsNeedsInCourtContent.errorMessageTypOfChair}")`,
+        1,
+      ),
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${Selectors.a}:text-is("${ReasonableAdjustmentsNeedsInCourtContent.errorMessageOther}")`,
+        1,
+      ),
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${Selectors.GovukErrorMessageCitizen}:text-is("${ReasonableAdjustmentsNeedsInCourtContent.errorMessageOther}")`,
+        1,
+      ),
+    ]);
+    await page.click(CheckListUniqueSelectors.parkingSpace);
+    await page.click(CheckListUniqueSelectors.differentTypeOfChair);
+    await page.click(CheckListUniqueSelectors.other);
+  }
+
+  private static async fillInFields({
+    page: page,
+    yesNoReasonableAdjustments: yesNoReasonableAdjustments,
+  }: Partial<ReasonableAdjustmentsNeedsInCourtPageOptions>): Promise<void> {
+    if (!page) {
+      throw new Error();
+    }
+    if (yesNoReasonableAdjustments) {
+      for (const selector of Object.values(CheckListUniqueSelectors)) {
+        await page.click(selector);
+      }
+      await Helpers.checkGroup(
+        page,
+        3,
+        ReasonableAdjustmentsNeedsInCourtContent,
+        "hiddenGovUkLabel",
+        Selectors.GovukLabel,
+      );
+      const textToFill: [string, string, string] = [
+        "loremIpsumParkingSpot",
+        "loremIpsumTypeOfChair",
+        "loremIpsumOther",
+      ];
+      for (let key of textToFill) {
+        let inputKey = key as keyof typeof TextboxInputUniqueSelectors;
```

Comment:
I think the keys in the array `textToFill` don't match the keys in the enum `TextboxUniqueSelectors`